main.go

```diff
@@ -74,10 +76,15 @@ func main() {
 		Namespace:              os.Getenv("WATCH_NAMESPACE"),
 		LeaderElectionID:       "b648e39a.open-cluster-management.io",
 	})
+
 	if err != nil {
 		setupLog.Error(err, "unable to start manager")
 		os.Exit(1)
 	}
+	if err := addonv1alpha1.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "unable to add to scheme")
+		os.Exit(1)
```

Comment:
Can we avoid exiting on this error?  The operator will be significantly degraded, but we can still perform the install activities.  Is there any possible way to recover from this error with a retry?

Reply:
@jlpadilla Addressed this in the latest change - could you please review it ?